### DISCUSSION TITLE
[Feature request]:How can I install OpenTOPAS on slurm cluster? (CentOS)

### DISCUSSION THREAD

**Discussion by DararaiPakraypet** [OpenTOPAS/OpenTOPAS#45] [Feature-Request] [answered]
### Feature request

Did you have any instruction for installation of openTOPAS on slurs cluster (CentOS)?

### TOPAS category

Geometry

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by icmeyer** [comment, marked as answer]
Hi,
The instructions should be very similar to the Debian quickstart guide, except instead of install the packages using `apt` will either have to use your clusters package configuration. Here is an example of a compilation script for a system that uses the `lmod` module system

```
#!/bin/bash

BUILDDIR="build"
INSTALLDIR="../install"
cd $BUILDDIR

# Cluster module handling
module purge
module load gdcm/2.6.8
module load cmake/3.19.2
module list

echo $(which gcc)
echo $(which g++)
export CXX=`which g++`
export CC=`which gcc`

export TOPAS_G4_DATA_DIR=/path/to/data
export Geant4_DIR=/path/to/geant4/install
cmake ../ -DCMAKE_INSTALL_PREFIX=$INSTALLDIR
make -j12 install
```

Geant4 can be installed in a similar fashion. Of course, all the modules and paths will be specific to your configuration. If this is all pretty new to you, I recommend reaching out to someone experienced with your cluster for help.
